test/e2e/users_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2020-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
+
+//+build e2e
+
+package e2e
+
+var correctUsers map[string]string
+
+func init() {
```

Comment:
Oh, no, haha, this is really stupid actually

I was fighting with gopls picking up these changes because of the build tag and I moved this into `init` from being a direct initialization because I was at that stage in debugging where you question your sanity and think "well maybe if I change this to the equivalent thing, it'll work, and then I can figure out why I thought the first thing was right in the first place?"

Anyway, then I fixed gopls and everything worked and I just forgot to change this to a direct initialization, especially since there's no real difference from a runtime perspective.

Thanks for listening to my story; I'll move this to a direct initialization :joy: